docs/README.md

```diff
@@ -18,7 +18,7 @@ npm install service-mocker-polyfills --save-dev
 
 ## Hello new world
 
-<p class="warning">Before starting, make sure you are using a develop server (e.g. `webpack-dev-server`) to serve your static assets. Assuming we are hosting our app in `http://localhost:3000`:</p>
+<p class="warning">Before starting, make sure you are using a develop server (e.g. <code>webpack-dev-server</code>) to serve your static assets. Assuming we are hosting our app in <code>http://localhost:3000</code>:</p>
```

Comment:
Ping @egoist, markdown syntax inside HTML tags [seems not to work with GitHub itself](https://github.com/service-mocker/service-mocker/blob/develop/docs/README.md#hello-new-world)